source/sync/event-library.txt

```diff
@@ -1,465 +0,0 @@
-.. _app-services-event-library:
-
-=============
-Event Library
```

Comment:
This should definitely _not_ be deleted. There used to be a link to it from the Device Sync ToC - it looks like we lost this from the ToC during the Sync section refactor: https://github.com/mongodb/docs-app-services/pull/191

Maybe a good place for this to live would be "App Builders Resources"?

Reply:
At least this proves the script will reveal mistakes like that!